src/components/SearchBar/SearchBar.tsx

```diff
@@ -217,6 +217,7 @@ export const SearchBar: React.FC<any> = chakra(
         invalidText={invalidText}
         isInvalid={isInvalid}
         ref={ref}
+        sx={{ containerType: "inline-size" }}
```

Comment:
It seems to me that container queries are so new that Chakra does not support them as style props. Is that true?

This doesn't have to be part of this ticket but we should update all components so that devs can do `<Component wrapper containerType="inline-size" ...>`. It might mean having a global base interface with the container query attributes and _all_ component interfaces inherit from it. Maybe? This is a nice-to-have update.